tools/rna_tools/cmv/cmcv.xml

```diff
@@ -0,0 +1,72 @@
+<tool id="cmcv" name="cmcv" version="@VERSION@.1">
+    <description>
+        CMCV is a tool for visualization of comparisons between RNA family models (CM)
+    </description>
+   <macros>
+        <import>macros.xml</import>
+    </macros>
+    
+    <expand macro="requirements" />
+
+    <expand macro="stdio" />
+    
+    <expand macro="version" />
+    <command><![CDATA[
+    CMCV
+    @COMMON_ARGS@
+    -m $model_filepath
+    -r $cmcompare_file
+    -p `pwd`
+    > 'warnings'   
+    ]]></command>
+    
+    <inputs>
+      <param name="model_filepath" type="data" format="cm" label="Input model"/>
+      <param name="cmcompare_file" type="data" format="cmcompare" label="Input comparison"/>
+      <section name="common" title="Common parameters">
+        <expand macro="common_parameters" />
+      </section>
+    </inputs>
+    
+    <outputs>
+      <collection name="output_pdfs" type="list" label="${tool.name} on ${on_string}: ">
+        <discover_datasets pattern="^[a-zA-Z0-9_-]*.pdf$" format="pdf"/>
+         <filter>output_format=='pdf'</filter>
+      </collection>
+      <collection name="output_svg" type="list" label="${tool.name} on ${on_string}: ">
+        <discover_datasets pattern="^[a-zA-Z0-9_-]*.svg$" format="svg"/>
+         <filter>output_format=='svg'</filter>
+      </collection>
+      <collection name="output_png" type="list" label="${tool.name} on ${on_string}: ">
+        <discover_datasets pattern="^[a-zA-Z0-9_-]*.png$" format="png"/>
+         <filter>output_format=='png'</filter>
+      </collection>
+      <collection name="output_ps" type="list" label="${tool.name} on ${on_string}: ">
+        <discover_datasets pattern="^[a-zA-Z0-9_-]*.ps$" format="ps"/>
+         <filter>output_format=='ps'</filter>
+      </collection>	
+    </outputs>
+    
+    <tests>
+      <test>
+        <param name="model_filepath" value="hammerheadClan.cm" />
+	<param name="hmmcompare_file" value="input.cmcompare" />
+        <output name="out1" file="Hammerhead_II.pdf" />
```

Comment:
the test will fail, because you output a collection, but only compare here on file ... you also expect here a collection.
Here is a nice example of filer+ collection + tests:
https://github.com/galaxyproject/tools-iuc/blob/ea40e3d84e7850eb4226d6c85f709dcad18d4ba9/tools/mothur/split.abund.xml#L120